release/release_data_tests.yaml

```diff
@@ -0,0 +1,910 @@
+###############
```

Comment:
Just to confirm this is just a copy paste from the existing `release_tests.yaml`? Assuming I don't need to review any of this.

Reply:
Yeah. Copy and paste of the "Batch inference" and "Data tests" sections